lib/memery.rb

```diff
@@ -64,5 +67,11 @@ module InstanceMethods
     def clear_memery_cache!
       @_memery_memoized_values = {}
     end
+
+    private
+
+    def monotonic_clock
```

Comment:
I don’t think that adding this method in every object that incudes Memery is a good idea. You can move it into Memery singleton class so that it lives only there.

Reply:
Good point. Done with wrapping module methods in `class << self` to avoid `def self.` repetition.